### PR TITLE
Added support for collecting ApiMappings from all desired modules

### PR DESCRIPTION
**Pull request by umbreak**
Fixes https://github.com/BlueBrain/nexus/issues/2086

### CODE REVIEW

File: delta/app/src/test/scala/ch/epfl/bluebrain/nexus/delta/routes/ResolversRoutesSpec.scala [hunk 1]
```diff
@@ -52,20 +52,22 @@ class ResolversRoutesSpec
   private val asAlice = addCredentials(OAuth2BearerToken(alice.subject))
   private val asBob   = addCredentials(OAuth2BearerToken(bob.subject))
 
-  private val org      = Label.unsafe("org")
-  private val am       = ApiMappings(Map("nxv" -> nxv.base, "Person" -> schema.Person))
-  private val projBase = nxv.base
-  private val project  =
+  private val org                = Label.unsafe("org")
+  private val defaultApiMappings = ApiMappings("_" -> schemas.resources, "resource" -> schemas.resources)
```
Comment (imsdu) [inline delta/app/src/test/scala/ch/epfl/bluebrain/nexus/delta/routes/ResolversRoutesSpec.scala:56]: We could use `Resources.mappings here no` ?

File: delta/app/src/test/scala/ch/epfl/bluebrain/nexus/delta/routes/ResolversRoutesSpec.scala [hunk 2]
```diff
@@ -52,20 +52,22 @@ class ResolversRoutesSpec
...
     ProjectGen.project("org", "project2", uuid = uuid, orgUuid = uuid, base = projBase, mappings = am)
 
   private val (_, projects) = {
     implicit val subject: Subject = Identity.Anonymous
     ProjectSetup
       .init(
         orgsToCreate = List(org),
-        projectsToCreate = List(project, project2)
+        projectsToCreate = List(project, project2),
+        defaultApiMappings = defaultApiMappings
```
Comment (imsdu) [inline delta/app/src/test/scala/ch/epfl/bluebrain/nexus/delta/routes/ResolversRoutesSpec.scala:70]: And put Resources.mappings as a default value here as it seems that is what we use during the tests when it does not really matter

**Review by imsdu — CHANGES_REQUESTED** [review]

**Review by bogdanromanx — CHANGES_REQUESTED** [review]
Looks good, but I think we should use a plural alias everywhere:
`file` -> `files`
`archive` -> `archives`
etc.

**Comment by umbreak** [comment]
No. Those are the schema aliases as in 1.4.x and get used in the following context (for example):
v1/resources/{org}/{proj}/`{schema}`/{id}

**Comment by bogdanromanx** [comment]
if these are the aliases we're using in v1.4 we need to keep them, but since the segment is used as an identifier for a collection of resources it should be plural (that's why we have "documents" and not "document"); should we also add their plurals then?

**Comment by umbreak** [comment]
We could. However I don't want to have a nightmarish API again where you have 100 ways of accessing the same endpoints.

**Review by bogdanromanx — APPROVED** [review]

**Pull request merged**